src/main/resources/application.properties

```diff
@@ -0,0 +1,12 @@
+#server
+server.port=7002
+#datasources
+spring.jpa.generate-ddl=true
+spring.datasource.url=jdbc:postgresql://138.68.181.72:5432/StreamForge
```

Comment:
look, pls dont use open creds data here
use them as by the link above, thanks

https://github.com/StreamForge/StreamForgeDB/blob/master/src/main/resources/application.properties